store/modules/forecastViz.js

```diff
@@ -131,7 +119,37 @@ export default (moduleOptions) => ({
     unselect_all_models (state) {
       state.current_models = state.default_models
       state.all_models = false
+    },
+    set_options(state, options){
+      console.log('xx2 set_options() called with options:', options)
+      state.target_variables = options.target_variables
+      state.target_var = options.init_target_var
+      state.locations = options.locations
+      state.location = options.init_location
+      state.intervals = options.intervals
+      state.interval = options.init_interval
+      state.available_as_ofs = options.available_as_ofs
+      state.as_of_date = options.init_as_of_date
+      state.current_date = options.current_date
+      state.models = options.models
+      state.current_models = options.default_models
+      state.default_models = options.default_models
+      state.colours = Array(parseInt(options.models.length / 10, 10) + 1).fill([
+        '#0d0887',
+        '#46039f',
+        '#7201a8',
+        '#9c179e',
+        '#bd3786',
+        '#d8576b',
+        '#ed7953',
+        '#fb9f3a',
+        '#fdca26',
+        '#f0f921'
+      ]).flat()
+      state.disclaimer = options.disclaimer
```

Comment:
previously, we were also initializing a variable called `all_models`
```suggestion
      state.all_models = options.all_models
      state.disclaimer = options.disclaimer
```